src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -513,10 +514,27 @@ protected override void Generate(RenameTableOperation operation, [CanBeNull] IMo
                     .Append(method);
             }
 
-            builder
-                .Append(" (")
-                .Append(ColumnList(operation.Columns))
-                .Append(")");
+            if (operators == null) {
+                builder
+                    .Append(" (")
+                    .Append(ColumnList(operation.Columns))
+                    .Append(")");
+            }
+            else {
+                var operatorList = operators.Split(' ');
+                var columnList = new List<string>();
+                for (var k = 0; k < operation.Columns.Count - 1; ++k) {
+                    var columnWithOperator = ColumnList(operation.Columns[k]);
+                    if (k < operatorList.Count) {
```

Comment:
It should also be possible to have a null in the operator list to skip explicitly defining the operator class for the correspond column (leaving it at the default).